src/picosvg/svg_transform.py

```diff
@@ -77,23 +78,50 @@ def tostring(self):
             return f'translate({", ".join(ntos(v) for v in self.gettranslate())})'
         return f'matrix({" ".join(ntos(v) for v in self)})'
 
-    @staticmethod
-    def product(first: "Affine2D", second: "Affine2D") -> "Affine2D":
-        """Returns the product of first x second.
+    def __matmul__(self, other: "Affine2D") -> "Affine2D":
+        """Returns the product of first × second. Order matters.
```

Comment:
I think here you may want to spell out the order to use to achieve "first do this then do that" semantics

Reply:
but we do already have `compose_ltr` for that, which already has some explanantion why it is useful..

Reply:
done https://github.com/googlefonts/picosvg/pull/227/commits/928409fbdab3b2ab57d48b1e228ca2ae89eb3d02